integration-test/src/features/concatenate-CRUD-by-file-upload.feature

```diff
@@ -0,0 +1,25 @@
+Feature: concatenate CRUD by file upload
+
+  Background:
+    Given GPD-Upload running
+
+  Scenario: Create and Update same Debt Position by file upload OK
+  // todo
+
+  Scenario: Create and Delete same Debt Position by file upload OK
+  // todo
+
+  Scenario: Create, Update and Delete same Debt Position by file upload OK
```

Comment:
why this scenario isn't cover by previous one ?

Reply:
in the view of a test based on [Black Box State Transition Testing](https://theqalead.com/topics/black-box-testing-techniques/) it is a different path not covered by the previous one